.codegen/service.py.tmpl

```diff
@@ -101,7 +101,7 @@ class {{.PascalName}}{{if eq "List" .PascalName}}Request{{end}}:{{if .Descriptio
 {{- end -}}
 
 {{range .Services}}
-class {{.Name}}API:{{if .Description}}
+class {{.PascalName}}API:{{if .Description}}
```

Comment:
Highlight

Reply:
OK, as this is aligned with the treatment of this in Go SDK.